src/__tests__/rerender.js

```diff
@@ -0,0 +1,46 @@
+import '@testing-library/jest-dom'
+import {defineComponent, h, computed} from 'vue'
+import {render} from '@testing-library/vue'
+import NumberDisplay from './components/NumberDisplay'
+
+test('calling rerender remounts the component and updates the props', () => {
```

Comment:
I would keep the suggestion to avoid using this method if it's not strictly necessary.

```suggestion
// It'd probably be better if you test the component that's doing the rerendering
// to ensure that the rerendered component is being updated correctly.
// That said, if you'd prefer to, for example, update the props of a rendered
// component, this function can be used to do so. 
test('calling rerender remounts the component and updates the props', () => {
```

Reply:
Done! I put the comment above the `test` line to be more like the old file if that's okay.